torchao/dtypes/affine_quantized_tensor.py

```diff
@@ -157,7 +158,16 @@ def dequantize(self, output_dtype=None):
 
     @staticmethod
     def _quantized_linear_op(input_tensor, weight_tensor, bias):
```

Comment:
where does _quantized_linear_op get called now? The code flow feels much harder to follow now.

Reply:
yeah this is not called in AffineQuantizedTensor right now. I see, OK let me think about how we can make this work then